frame.go

```diff
@@ -441,6 +443,18 @@ func newFramer(r io.Reader, w io.Writer, compressor Compressor, version byte) *f
 	return f
 }
 
+func newFramerWithExts(r io.Reader, w io.Writer, compressor Compressor, version byte,
+	cqlProtoExts []cqlProtocolExtension) *framer {
+
+	f := newFramer(r, w, compressor, version)
+
+	if lwtExt := findCQLProtoExtByName(cqlProtoExts, lwtAddMetadataMarkKey); lwtExt != nil {
+		f.flagLWT = (*lwtExt).(lwtAddMetadataMarkExt).lwtOptMetaBitMask
```

Comment:
This can blow up right? In case someone adds another thing implementing the interface this conversion blows up. 
Conversions in Go returns a complementary boolean that can be used to ensure panic free conversions.

Reply:
In theory, it can, but is unlikely to happen since every implementation of the interface will have its own string constant, which identifies the extension. But I can add one more check for it, if you feel more comfortable with that.

Reply:
I know these are in the category "I know the types being passed around" but this happened to me before in a long living project so I think handling it makes it both safer and more clear even if the probability of the error is very low.